examples/voice-assistant/minimal_assistant.py

```diff
@@ -29,10 +29,10 @@ async def entrypoint(ctx: JobContext):
     await assistant.say("Hey, how can I help you today?", allow_interruptions=True)
 
 
-async def request_fnc(req: JobRequest) -> None:
+async def job_request_fnc(req: JobRequest) -> None:
     logging.info("received request %s", req)
-    await req.accept(entrypoint)
+    await req.accept()
 
 
 if __name__ == "__main__":
-    cli.run_app(WorkerOptions(request_fnc))
+    cli.run_app(WorkerOptions(job_request_fnc=job_request_fnc, job_entrypoint_fnc=entrypoint))
```

Comment:
having `job_` prefix seems redundant? also, could it be left blank to auto accept?

```suggestion
    cli.run_app(WorkerOptions(entrypoint_fnc=entrypoint))
```

Reply:
we can discuss more about it tmrw